modules/tools/AppFacCLI/bin/build

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+echo -e "\nGenerating Binary..."
+
+ROOT_DIR=$(cd $(dirname $(dirname $0)) && pwd)
+
+CLI_GOPATH=$ROOT_DIR/tmp/cli_gopath
+rm -rf $CLI_GOPATH
+mkdir -p $CLI_GOPATH/src/github.com/Dilhasha/AppFacCLI
+ln -s $ROOT_DIR $CLI_GOPATH/src/github.com/Dilhasha/AppFacCLI/cli
+
+GODEP_GOPATH=$ROOT_DIR/Godeps/_workspace
+
+GOPATH=$CLI_GOPATH:$GODEP_GOPATH:$GOPATH go build -o $ROOT_DIR/out/appfac ./main
+rm -rf $CLI_GOPATH
+
```

Comment:
Please add the golang dependencies here (go get). Otherwise people can't build this.